pcdsdevices/usb_encoder.py

```diff
@@ -27,6 +27,26 @@ class USBEncoder(BaseInterface, Device):
 
     tab_whitelist = ['pos', 'set_zero', 'scale', 'offset']
 
+    def __init__(self, stage=None):
+        self.stage = stage
```

Comment:
A few things here:
* `stage` is a reserved term in bluesky - you're overriding an existing method by doing this
* `__init__` arguments aren't correct for a `Device`:
    * You'll need to have `prefix` and the usual `__init__` arguments
    * You'll also need to call `super().__init__(args_and_kwargs_here)` in the body

Reply:
Hi Ken, the code was not quite yet ready for review, sorry about that.
I am going to replace the `stage` by something else, thanks for letting me know

Reply:
No problem - I saw you requesting reviews so assumed otherwise :)